t/v27_features.py

```diff
@@ -20,3 +20,92 @@
 assert(result['uri'] == '/issue_120/{}'), "expected result"
 print ('[+] OK')
 
+preset_method_location = BASE_URL + '/skipping_test'
+body = [
+  {
+    "skipping_map_at_the_beginning": {
+      "ttl": 123
+    },
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "id": 1,
+    "skipping_map_in_the_middle": {
+      "ttl": 123
+    },
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "skipping_map_in_the_end": {
+      "ttl": 123
+    }
+  },
+  {
+    "skipping_array_at_the_beginning": [
+      "ttl",
+      123
+    ],
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "meta": [
+      "ttl",
+      123
+    ]
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "skipping_array_in_the_middle": [
+      "ttl",
+      123
+    ],
+    "params": []
+  },
+  {
+    "id": 1,
+    "method": "skipping_test",
+    "params": [],
+    "skipping_array_in_the_end": [
+      "ttl",
+      123
+    ]
+  },
+  {
+    "skipping_map_with_nesting": {
+      "ttl": {
+        "ttl": 123,
+        "array":[]
+        }
+    },
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  },
+  {
+    "skipping_array_with_nesting": [
+      "ttl",
+      [
+        "ttl",
+        {
+          "map":
+          123
+        }
+      ]
+    ],
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  }
+]
+print ('[+] Garbage fields skipping.')
+result = post_success_pure(preset_method_location, body, {})
+for item in result:
+  assert (item['result'][0] == 200), "response in not 200"
```

Comment:
I would want to verify not only that a request encoding becomes correct, but also that all data we send were passed to tarantool. Please, return passed data back and verify here.

Reply:
Done.

Reply:
Not exacly. See https://github.com/tarantool/nginx_upstream_module/pull/138#discussion_r583066001